src/Custom-sample.h

```diff
@@ -107,8 +107,18 @@
                                                                             //   8 = Generic HTTP
                                                                             //   9 = FHEM HTTP
 
+#ifdef ESP8266
 #define DEFAULT_PIN_I2C_SDA                     4
+#endif
+#ifdef ESP32
+#define DEFAULT_PIN_I2C_SDA                     21
```

Comment:
Maybe we should set this to -1 on ESP32.
There is no "standard" I2C pin for the boards out there.

Reply:
OK